File: 05. Percabangan/F. Floor dan Ceiling/f_floor_dan_ceiling.pas
program f_floor_dan_ceiling;
uses crt;
var
N : real;
begin
  readln(N);
  if (N >= 0) then begin
    if (N > trunc(N)) then
      writeln(trunc(N), ' ', trunc(N)+1)
    else
      writeln(trunc(N), ' ', trunc(N));
  end
  else begin
    if (N < trunc(N)) then
      writeln(trunc(N)-1, ' ', trunc(N))
    else
      writeln(trunc(N), ' ', trunc(N));
  end;
end.

File: 05. Percabangan/F. Floor dan Ceiling/f_floor_dan_ceiling.py
import math

N = float(input())
if (N >= 0):
    if (N > math.trunc(N)):
        print(math.trunc(N), math.trunc(N)+1)
    else:
        print(math.trunc(N), math.trunc(N))
else:
    if (N < math.trunc(N)):
        print(math.trunc(N)-1, math.trunc(N))
    else:
        print(math.trunc(N), math.trunc(N))
